### DISCUSSION TITLE
How to get the DeltaTime when using the flowcaht control?

### DISCUSSION THREAD

**Discussion by WarmBed** [TOSUN-Shanghai/TSMaster#101] [Q&A] [answered]
As title, I'm trying to calute some value about it will need the flowchat detla time. How can I get it?

如抬頭，如何再用圖形程式時獲取detla time來進行我所需要的計算?

Thanks

**Comment by freshhope** [comment, marked as answer]
![image](https://github.com/TOSUN-Shanghai/TSMaster/assets/16358754/c048f3c4-8ec3-48dd-9047-24da19acc513)

import this program:
[2023_06_01_13_47_05.zip](https://github.com/TOSUN-Shanghai/TSMaster/files/11621425/2023_06_01_13_47_05.zip)

**Reply by WarmBed** [reply]
感謝 馬上測試 請問有地方會描述小程序的應用嗎?

**Comment by freshhope** [comment]
![image](https://github.com/TOSUN-Shanghai/TSMaster/assets/16358754/06eed87b-089c-415b-beaf-bffc0b1de86f)

**Reply by WarmBed** [reply]
![image](https://github.com/TOSUN-Shanghai/TSMaster/assets/72774359/46f9e363-10c2-4a3e-8350-27295ac92544)

![image](https://github.com/TOSUN-Shanghai/TSMaster/assets/72774359/6a64ab3c-3dbd-4eb0-9b76-f41e9d6b4059)

Hi 我描述的不太清楚，是關於這部分 每個app是甚麼功能，並且裡面的每一項需要填寫甚麼 請問在哪能看到呢  謝謝

**Reply by freshhope** [reply]
![image](https://github.com/TOSUN-Shanghai/TSMaster/assets/16358754/ef3abc66-e9c0-46ee-9a7b-9063ec3a2968)

**Reply by WarmBed** [reply]
@freshhope 感謝 大概懂怎看了  但我想讀取excel並寫入一些資料 該怎執行比較好呢?

![image](https://github.com/TOSUN-Shanghai/TSMaster/assets/72774359/3b83d1c0-40d3-40bb-8c9c-d2815f1aaaf2)


另外我在"系統消息"需要一直廣播多項資料，是否用C 小程序寫一個print好呢? 目前是用下圖方式進行

![TSMaster_HFO79HJcgi](https://github.com/TOSUN-Shanghai/TSMaster/assets/72774359/79d67c6d-f5ed-4624-96c4-2fea8e094d81)

**Comment by WarmBed** [comment]
@freshhope 
早安，我想詢問，當我新增了一個判斷條件在循環裡面，時間整個計算都跑掉了，有甚麼辦法能進行修正嗎?

謝謝
![image](https://github.com/TOSUN-Shanghai/TSMaster/assets/72774359/0f1dc274-14ee-457a-aeb5-9e3e14f614db)
![image](https://github.com/TOSUN-Shanghai/TSMaster/assets/72774359/4c560899-b9d8-436a-b54d-ce5f329a41fa)
